extensions/sql-migration/package.nls.json

```diff
@@ -8,6 +8,7 @@
 	"start-migration-command": "Migrate to Azure SQL",
 	"send-feedback-command": "Feedback",
 	"new-support-request-command": "New support request",
+	"refresh-migrations-command": "Refresh",
```

Comment:
Please remove this resource as well if you do not plan on using it.

Reply:
This is used. a Refresh button/command I am adding to the commands list for the dashboard

Reply:
This is used. a Refresh button/command I am adding to the commands list for the dashboard